mm2src/mm2_core/src/mm_ctx.rs

```diff
@@ -776,6 +815,13 @@ impl MmCtxBuilder {
                         .expect("Invalid json value in 'event_stream_configuration'.");
                 ctx.event_stream_configuration = Some(event_stream_configuration);
             }
+
+            let healthcheck_config = &ctx.conf["healthcheck_config"];
+            if !healthcheck_config.is_null() {
+                let healthcheck_config: HealthcheckConfig =
+                    json::from_value(healthcheck_config.clone()).expect("Invalid json value in 'healthcheck_config'.");
```

Comment:
@onur-ozkan  shouldn't we result to using default here if serde fails? since we have Default implemented ?

Reply:
I think if there is invalid value on config field that should be reported as a feedback to user. It's also `serde`'s default behaviour (try to give integer value on boolean field with `default` attribute).